annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/index/Error.java

```diff
@@ -46,9 +46,9 @@ public class Error {
   /** Error message. */
   public final String message;
   /** The fixes which can resolve this error (possibly empty). */
-  private final ImmutableSet<Fix> resolvingFixes;
+  protected final ImmutableSet<Fix> resolvingFixes;
```

Comment:
Just curious, should the `Error` class be abstract now?  Like do we expect each checker to have its own concrete error class?

Reply:
At first I thought a checker can just use the default `Error` class, but it would make more sense to expect each checker have their own concrete error class (even if it is just extending the base class). Made it abstract. 6c1a5dc739f288e54754df3f4fc50ca7beb93e8a